dolphin/task_manager/manager.py

```diff
@@ -71,3 +71,13 @@ def sync_storage_resource(self, context, storage_id, resource_task):
         cls = importutils.import_class(resource_task)
         device_obj = cls(context, storage_id)
         device_obj.sync()
+
+    def remove_storage_resource(self, context, storage_id, resource_task):
+        cls = importutils.import_class(resource_task)
+        device_obj = cls(context, storage_id)
+        device_obj.remove()
+
+    def remove_storage_in_memory(self, context, storage_id, resource_task):
+        cls = importutils.import_class(resource_task)
+        device_obj = cls(context, storage_id)
+        device_obj.remove()
```

Comment:
how about remove_storage_in_cache?
it's seems resource_task is not required here

Reply:
Done